src/server/active_tran_server.hpp

```diff
@@ -78,7 +78,7 @@ class active_tran_server
     std::string m_ps_hostname;
     int m_ps_port = -1;
 
-    std::unique_ptr<page_server_conn_t> m_page_server_conn;
+    std::vector<std::unique_ptr<page_server_conn_t>> m_page_server_conn;
```

Comment:
I would change the name of the member to smth that suggests it is a collection and not a single connection:
```suggestion
    std::vector<std::unique_ptr<page_server_conn_t>> m_page_server_conn_vec;
```
is my preference. Yours might be different. :)

Reply:
a proper name for a proper field.